app/views/comment_mailer/notify_note_author.html.erb

```diff
@@ -19,6 +19,15 @@ Do NOT reply to this email; click this link to respond:
 
 <hr />
 
+<p>Look like spam?
+  <% if @user.can_moderate? %>
```

Comment:
Hi @qm3, could you please change this line to `<% if @user.user.can_moderate? %>` ? That will resolve the errors listed by plotsbot. Thanks!